CDCgov/prime-reportstream
# Victor/8320 FHIR translation AOE questions

This PR adds the ability to map date values for AOE questions

Test Steps:
1. Run integration tests and make sure they pass.
2. Run: ./prime fhirdata --input-file src/testIntegration/resources/datatests/FHIR_to_HL7/sample_SR_1_20230301-0001.fhir -s metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml --output-format HL7 --output-file src/testIntegration/resources/datatests/FHIR_to_HL7/sample_AOE_1_20230301-0001.hl7 

## Changes
- Update FHIR to HL7 mappings

## Checklist

### Testing

## Linked Issues
- Fixes #8320
